src/client/WebClient.ts

```diff
@@ -266,6 +269,27 @@ export class WebClient extends Client {
 		}
 	}
 
+	async openLivechatRoom(rid: string): Promise<void> {
+		const end = prom.openRoom.startTimer();
+		const endAction = prom.actions.startTimer({ action: 'openRoom' });
+		try {
+			const calls: Promise<unknown>[] = [
+				this.client.methodCall('loadHistory', rid, null, 50, new Date()),
+			];
+
+			calls.push(this.client.methodCall('getRoomRoles', rid));
+
+			await Promise.all(calls);
```

Comment:
I was expecting to have `getVisitorInfo` among this `calls` .. they always happen together.. is there any other call that should be added here?